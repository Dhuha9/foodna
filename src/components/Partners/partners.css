.partners-sections {
  width: 80%;
  height: 266px;
  margin: 0 auto;
}

.partner-image {
  height: 80px;
  width: 80px;
  border-radius: 50%;
  margin-bottom: 20px;
}

.partners-title {
  font-family: 'Roboto', sans-serif;
  font-size: large;
  padding: 30px;
  text-align: center;
}

.partners-grid {
  display: grid;
  grid-template-columns: 120px 120px 120px 120px 120px 120px;
  justify-content: center;
}

@media only screen and (max-width: 682px) {
  .partners-grid {
    display: grid;
    grid-template-columns: 120px 120px 120px 120px 120px;
    justify-content: center;
  }
}

@media only screen and (max-width: 592px) {
  .partners-grid {
    display: grid;
    grid-template-columns: 120px 120px 120px 120px;
    justify-content: center;
  }
}

@media only screen and (max-width: 480px) {
  .partners-grid {
    display: grid;
    grid-template-columns: 120px 120px 120px;
    justify-content: center;
  }
}
