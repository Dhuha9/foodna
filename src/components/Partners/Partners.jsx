import React, { Component } from 'react';
import Partner from '../Partner/Partner';
import partnerData from '../../partner.json';
import './partners.css';
export default class Partners extends Component {
  render() {
    const { t } = this.props;

    return (
      <div className="partners-section pb-5 ">
        <p className="partners-title">{t('PartnersComponent')}</p>
        <div className="partners-grid">
          {partnerData.map((partners, index) => {
            return <Partner key={index} img={partners.img} />;
          })}
        </div>
      </div>
    );
  }
}
