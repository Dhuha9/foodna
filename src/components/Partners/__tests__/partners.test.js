import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Partners from '../Partners';
it('renders apply now correctly', () => {
  const tree = renderer.create(<Partners />).toJSON();
  expect(tree).toMatchSnapshot();
});
