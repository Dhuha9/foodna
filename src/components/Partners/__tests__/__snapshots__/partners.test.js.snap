// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders apply now correctly 1`] = `
<div
  className="partners-section"
>
  <p
    className="partners-title"
  >
    Who Works With Us?
  </p>
  <div
    className="partners-grid"
  >
    <div>
      <img
        alt="sponsor"
        className="partner-image"
        src="sponsor.png"
      />
    </div>
    <div>
      <img
        alt="sponsor"
        className="partner-image"
        src="sponsor.png"
      />
    </div>
    <div>
      <img
        alt="sponsor"
        className="partner-image"
        src="sponsor.png"
      />
    </div>
    <div>
      <img
        alt="sponsor"
        className="partner-image"
        src="sponsor.png"
      />
    </div>
    <div>
      <img
        alt="sponsor"
        className="partner-image"
        src="sponsor.png"
      />
    </div>
    <div>
      <img
        alt="sponsor"
        className="partner-image"
        src="sponsor.png"
      />
    </div>
  </div>
</div>
`;
